### ISSUE TITLE
node crash creating 2 document with the same id

### ISSUE THREAD

**Issue by danilopao** [avital/sharedb-mingo-memory#8]
when using "sharedb-mingo-memory", if 2 clients attempt to create a document with the same id, there is an exception and node crashes. 
There is no crash in the same schenario, when using "sharedb-mongo".

Module versions:
    "sharedb": "1.0.0-beta.13"
    "sharedb-mingo-memory": "1.1.0"

This is the Exception:

TypeError: this.getOpCollection is not a function at ShareDBMingo.DB.getCommittedOpVersion (.\node_modules\sharedb-mongo\index.js:433:8) at .\node_modules\sharedb\lib\submit-request.js:66:20 at .\node_modules\sharedb\lib\db\memory.js:66:5 at _combinedTickCallback (internal/process/next_tick.js:131:7) at process._tickCallback (internal/process/next_tick.js:180:9)
message: "this.getOpCollection is not a function"

**Comment by ericyhwang** [comment]
Thanks for the report!

What's odd is that the class inheritance chain is ShareDBMingo -> sharedb.MemoryDB -> sharedb.DB, which doesn't involve sharedb-mongo at all. And yet, the stack trace there has a reference to `sharedb-mongo\index.js`.

**Comment by ericyhwang** [comment]
I think I've spotted the issue.

sharedb-mongo overwrites `getCommittedOpVersion` on the base `DB.prototype`, when it was supposed to define the function on its own prototype:
https://github.com/share/sharedb-mongo/blob/v1.0.0-beta.7/index.js#L431

Normally, that doesn't result in issues, when using just a single adapter. But if you happen to load sharedb-mongo and sharedb-mingo-memory in the same set of tests, that would trigger the issue.

@danilopao - I've prepped [a fix in sharedb-mongo, on a branch](https://github.com/share/sharedb-mongo/commit/7bd9c4415b6964165c878529f7e5542988b22885). Can you install sharedb-mongo at that commit, and let me know if that fixes the crash you're seeing?

```
npm install share/sharedb-mongo#7bd9c4415b6964165c878529f7e5542988b22885
```

**Comment by danilopao** [comment]
Your commit fixed my crash. Good to know it only happens when including both adapters.
Thanks.

**Comment by ericyhwang** [comment]
Fix is merged and published in [sharedb-mongo@1.0.0-beta.8](https://github.com/share/sharedb-mongo/releases/tag/v1.0.0-beta.8)

Thanks for verifying that the fix works!
